### DISCUSSION TITLE
Confusion about Pro Badge for Teachers/Professors who applied for GitHub Education

### DISCUSSION THREAD

**Discussion by qcbegin** [community/community#145777] [GitHub Education]
### Hi everyone,

I have a question regarding faculty benefits under the GitHub Education program. While the documentation states that faculty members can upgrade to the GitHub Team plan, I'd like to clarify whether this includes a Pro badge on their profile.
Specifically, do faculty members receive a Pro badge when approved for GitHub Education benefits? I've noticed that GitHub Team and Enterprise subscribers typically don't display Pro badges, unlike GitHub Pro plan subscribers, which makes this unclear.

Thank you for clarifying this distinction.

**Comment by swapnilsparsh** [comment]
@qcbegin You have to enable it from you profile settings which can be accessed from here - https://github.com/settings/profile
